core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -147,15 +155,14 @@ class WorkspaceService(
   val terraBucketWriterRole: String,
   rawlsWorkspaceAclManager: RawlsWorkspaceAclManager,
   multiCloudWorkspaceAclManager: MultiCloudWorkspaceAclManager,
-  val fastPassServiceConstructor: (RawlsRequestContext, SlickDataSource) => FastPassService,
+  val fastPassServiceConstructor: RawlsRequestContext => FastPassService,
   val workspaceRepository: WorkspaceRepository,
   val billingRepository: BillingRepository
 )(implicit protected val executionContext: ExecutionContext)
-    extends RoleSupport
```

Comment:
This no longer needs `RoleSupport`, since the admin functions were moved to a separate service.